exist-installer/src/main/izpack/install.xml

```diff
@@ -172,7 +172,7 @@
         <pack name="shared" required="true" preselected="true" parent="Apps">
```

Comment:
The package name should be changed also here and where it is referenced, i.e. `name="shared"` is no longer correct.

Reply:
Also `You should at least install the "dashboard" and the "shared" package` above is no longer correct - please revise that too.

Reply:
done